docs/design/storage-node-installer-windows.md

```diff
@@ -0,0 +1,63 @@
+# Storage Node Installer for Windows
+
+## Abstract
+
+This design doc outlines how we will complete an installation process for a storage node on the Windows operating system. 
+We need to ensure that updater binary starts on computer start-up, without logging into the system. 
+We will build an msi for installing the necessary files onto the operating system. 
+We will set up the installed binary to run as a Windows service.
+
+## Background
+
+Docker is currently used to maintain storage nodes.
+Docker however does not run on Windows, yet Windows is a popular operating system among potential storage node operators.
+We need to install the storage node onto the user's operating system and configure the operating system to run the storage node as a Windows service. 
+We also need to avoid triggering UAC so that the storage node process can run in the background while the user is logged out.
+
+## Design
+
+### MSI
+* Install [go-msi](https://github.com/mh-cbon/go-msi)
+* Create a wix.json file like [this one](https://github.com/mh-cbon/go-msi/blob/master/wix.json)
+* Apply a GUID with `go-msi set-guid`, you must do it once only for each app.
+* Run `go-msi make --msi your_program.msi --version 0.0.2`
+* MSI must install storage node binary.
+* MSI must install watch dog binary.
+* MSI must install Automatic Updater binary.
+
+### service register script
+* Modify storage node/automatic updater startup code to implement [ServiceMain](https://docs.microsoft.com/en-us/windows/win32/api/winsvc/nc-winsvc-lpservice_main_functiona).
```

Comment:
FWIW, I use https://nssm.cc/ for running the vanilla storagenode.exe as a Windows Service on my AcePC node. No need to implement `ServiceMain` this way.

Reply:
NSSM also features redirecting stderr and stdout to a file and rotating this output file.